### PR TITLE
Single player footer fix

### PR DESCRIPTION
**Pull request by StnsGeneral**
**Part of my ticket was to remove the footer in the single player cards. A different group has since updated the footer and made the footer more in line with the Figma. I don't think that removing the footer is going to be the direction that we are wanting to go. But it was in there, so it is completed.**

@derekjpeters What direction do we want for the footer ☝🏻

Changed the text when submitting a file from "Save" to "Submit"

Changed the "I'd rather draw" and "I'd rather write" buttons to be antd button components. They now stand out a lot more and are very obviously buttons.

After submitting a file, the submission field was not clearing. Added some logic that clears that now.

https://www.loom.com/share/5ad17e2913414bed936361c60434dfe3

https://trello.com/c/KVwUj2dd/714-single-player-gameplay-mission-write-see-checklist
https://trello.com/c/GF8uxVBQ/712-single-player-gameplay-mission-read-see-checklist
https://trello.com/c/YegIug3J/713-single-player-gameplay-mission-draw-see-checklist

### CODE REVIEW

**Comment by StnsGeneral** [comment]
[Single Player - gameplay/mission/draw 
SEE CHECKLIST](https://trello.com/c/YegIug3J/713-single-player-gameplay-mission-draw-see-checklist)

**Comment by StnsGeneral** [comment]
[Single Player - gameplay/mission/read SEE CHECKLIST](https://trello.com/c/GF8uxVBQ/712-single-player-gameplay-mission-read-see-checklist)

**Comment by StnsGeneral** [comment]
[Single Player - gameplay/mission/write 
SEE CHECKLIST](https://trello.com/c/KVwUj2dd/714-single-player-gameplay-mission-write-see-checklist)

File: src/components/pages/GamePlay/GamePlayMain.js [hunk 1]
```diff
@@ -68,7 +68,7 @@ function GamePlayMain(props) {
         <Route component={NotFoundPage} />
       </Switch>
 
-      <ChildFooter />
+      {/* <ChildFooter /> */}
```
Comment (PVigar88) [inline src/components/pages/GamePlay/GamePlayMain.js]: maybe @ashtilawat23 can comment if this is needed?  I thought there was a mention on this just being a minimal footer with basic info as compared to the more robust version for the footers in parent pages.

**Review by PVigar88 — CHANGES_REQUESTED** [review]
Aside from where I think Ash's comment would be useful, I think this looks good.

File: src/components/pages/GamePlay/GamePlayMain.js (hunk 1 above)
Comment (tinomen) [reply src/components/pages/GamePlay/GamePlayMain.js]: Don't leave commented out code, please

**Comment by bipin-shrestha** [comment]
The Loom video gave clear idea of all the changes made and everything looks good Tristan.

**Comment by derekjpeters** [comment]
I think the alignment is better.  We can remove this PR w/o merging.

**Pull request closed without merging**